spec/hyacinth/adapters/digital_object_search_adapter/solr/system_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'shared_examples/digital_object_search_adapter/shared_examples'
+
+describe Hyacinth::Adapters::DigitalObjectSearchAdapter::Solr, solr: true do
+  let(:adapter) { Hyacinth::Config.digital_object_search_adapter }
```

Comment:
If this test is testing the `Hyacinth::Adapters::DigitalObjectSearchAdapter::Solr` implementation, you'll want to instantiate a new solr adapter rather than relying on the adapter selected by `Hyacinth::Config.digital_object_search_adapter`.

Reply:
Added a comment here to refer to subsequent ticket - I agree, but there are some complications to deal with that seem out of scope of this ticket.

Reply:
👍